backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/TaskApi.java

```diff
@@ -14,47 +23,141 @@
 public class TaskApi {
 
   // todo, make jobs private to the user?
-  public static TaskService taskService = new TaskServiceInMemory();
+  public static TaskService taskService = new TaskServiceInDatabase();
+  // to schedule jobs, see MolgenisSessionManager how we keep this in sync with Database using a
+  // TableListener
+  public static TaskServiceScheduler taskSchedulerService = new TaskServiceScheduler(taskService);
```

Comment:
by injecting these you could write a unit test

Reply:
not really needed in my mind because this is not a varable, unless mocking is desired?